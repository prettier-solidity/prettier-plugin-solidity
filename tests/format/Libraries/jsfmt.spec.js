run_spec(__dirname, ['solidity-parse']);
run_spec(__dirname, ['solidity-parse'], { bracketSpacing: true });
