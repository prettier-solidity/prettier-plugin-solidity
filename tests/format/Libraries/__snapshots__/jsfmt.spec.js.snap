// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Libraries.sol - {"bracketSpacing":true} format 1`] = `
====================================options=====================================
bracketSpacing: true
parsers: ["solidity-parse"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
  using   Lib    for uint;
using Lib for
    uint global;
using { f} for uint;
using {f } for uint global;
using { a, very,     long, and, complex,     Function, list, with, long, names }     for
    uint;
using { a, very,
        long, and, complex, Function,
        list, with, long, names }       for uint global;

=====================================output=====================================
using Lib for uint256;
using Lib for uint256 global;
using { f } for uint256;
using { f } for uint256 global;
using {
    a,
    very,
    long,
    and,
    complex,
    Function,
    list,
    with,
    long,
    names
} for uint256;
using {
    a,
    very,
    long,
    and,
    complex,
    Function,
    list,
    with,
    long,
    names
} for uint256 global;

================================================================================
`;

exports[`Libraries.sol format 1`] = `
====================================options=====================================
parsers: ["solidity-parse"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
  using   Lib    for uint;
using Lib for
    uint global;
using { f} for uint;
using {f } for uint global;
using { a, very,     long, and, complex,     Function, list, with, long, names }     for
    uint;
using { a, very,
        long, and, complex, Function,
        list, with, long, names }       for uint global;

=====================================output=====================================
using Lib for uint256;
using Lib for uint256 global;
using {f} for uint256;
using {f} for uint256 global;
using {
    a,
    very,
    long,
    and,
    complex,
    Function,
    list,
    with,
    long,
    names
} for uint256;
using {
    a,
    very,
    long,
    and,
    complex,
    Function,
    list,
    with,
    long,
    names
} for uint256 global;

================================================================================
`;
